contrib/opencensus-ext-azure/opencensus/ext/azure/common/transport.py

```diff
@@ -42,6 +42,10 @@ def _transmit(self, envelopes):
         This function should never throw exception.
         """
         try:
+            proxies = None
```

Comment:
Do you need this line?

Reply:
I set proxies to None because running a `json.loads(None)`, if no proxies are passed in, results in an exception. Does that make sense?

Reply:
If `self.options.proxy` is None, are you able to change it to an empty dict string?

Reply:
Great suggestion, I tested that we're able to send a request with ```proxies=json.loads("{}")```. I will update the code to reflect that by setting the default proxies option to be ```"{}"```

Reply:
Awesome. Make sure to initialize it in `process_options` instead of having a default `{}` value for proxies.

Reply:
I found that I need to initialize that value to `"{}"` in the Options map because even if we set it to a dictionary in process options , that options.proxies values ends up being converted to a string - I found this when running the unit tests. 

So it seems we'd have to keep that json.loads behavior in transport.py to make sure that argument is correctly loaded as a dictionary.

Reply:
You don't set it to a dictionary in process options. You set it to a string with an empty dictionary like you've said: `"{}"`.